packages/uswds-core/src/styles/mixins/general/table.scss

```diff
@@ -217,36 +221,29 @@ $table-unsorted-icon-color: get-color-token-from-bg(
   margin: units(2.5) 0;
   text-align: left;
 
-  thead {
-    th {
-      background-clip: padding-box;
-      color: $table-header-text-color;
+  thead td,
+  tfoot td,
+  th {
+    background-clip: padding-box;
+    line-height: line-height($theme-body-font-family, $theme-input-line-height);
+    background-color: color($theme-table-header-background-color);
+    color: $table-header-text-color;
+    + tfoot {
```

Comment:
**issue**: Not seeing where this applies. The generated CSS doesn't make sense either. A `tfoot` wouldn't be a sibling of a `thead td`.

```css
/* uswds.css */
.usa-table thead td + tfoot,
.usa-table tfoot td + tfoot,
.usa-table th + tfoot{
  font-weight:700;
}
```

 Can we fix the selector or remove if not needed?